scripts/utils.sh

```diff
@@ -33,12 +37,13 @@ sync_csi_proxy_integration_tests() {
 run_csi_proxy_integration_tests() {
   echo "Run integration tests"
   local ps1=$(cat << 'EOF'
-  "& {
-    $ErrorActionPreference = \"Stop\";
-    .\integrationtests.test.exe --test.v
-  }"
+    $ErrorActionPreference = "Stop";
```

Comment:
did it fail when it was in its own scope e.g. when inside `&{ }`

Reply:
The error code doesn't propagate correctly when in a subshell. We want the overall command to fail if the test fails.